reference/enterprise-api-ref/host.markdown

```diff
@@ -126,6 +126,39 @@ future. Key purging usually take an effect within 5-10 minutes.
 of [`trustkeysfrom`][cf-serverd#trustkeysfrom] for the hub hosts may re-appear
 and resume being collected from after being deleted.
 
+## Hosts list grouped by hard classes
+
+**URI:** https://hub.cfengine.com/api/hosts/by-class
```

Comment:
Still feels to me like our api endpoints should be versioned.

Reply:
From my point of view, versioning makes no too much sense right now because all changes are backward compatible. we might add versioning when some breakable changes gonna be added. And having version as part of the URL isn't the best practice of versioning.